server/manager/src/main/java/org/apache/accumulo/manager/tableOps/tableImport/PopulateMetadataTable.java

```diff
@@ -68,7 +69,7 @@ class PopulateMetadataTable extends ManagerRepo {
   static void readMappingFile(VolumeManager fs, ImportedTableInfo tableInfo, String importDir,
       Map<String,String> fileNameMappings) throws Exception {
     try (BufferedReader in = new BufferedReader(
-        new InputStreamReader(fs.open(new Path(importDir, "mappings.txt")), UTF_8))) {
+        new InputStreamReader(fs.open(new Path(importDir, IMPORT_MAPPINGS_FILE)), UTF_8))) {
```

Comment:
I'm pretty sure that with multiple resources in a try-with-resources block, you need to separate their initialization in order for their `close()` to be called correctly (as opposed to creating an object in another objects constructor like whats going on here). So in this case creating the `InputStreamReader` object followed by a semicolon then creating the `BufferedReader` would allow each objects `close` to be called correctly. I know you didn't set it up like this in this PR but might as well fix it while editing the same line.

Reply:
Yeah, I think it depends on implementation but it doesn't hurt to separate them.